rpc_util.go

```diff
@@ -502,4 +502,12 @@ const (
 // Version is the current grpc version.
```

Comment:
Could you please update this to:

```go
// Version is the current grpc-go release version.
```

This should help avoid confusion with the new consts.

Reply:
Done!